Libplanet/Action/AccountStateDeltaExtensions.cs

```diff
@@ -32,6 +32,16 @@ internal static class AccountStateDeltaExtensions
                 )
             ).ToImmutableDictionary();
 
+        internal static IImmutableDictionary<Currency, FungibleAssetValue>
+            GetUpdatedTotalSupplies(this IAccountStateDelta delta) =>
+            (
+                from currency in delta.TotalSupplyUpdatedCurrencies
+                where currency.TotalSupplyTrackable
+                let totalSupply = delta.GetTotalSupplyImpl(currency)
+                where totalSupply.HasValue
+                select new KeyValuePair<Currency, FungibleAssetValue>(currency, totalSupply.Value)
```

Comment:
It's never justified, but we haven't used SQL-style LINQ expressions. It might be better to keep avoiding it for consistency, but I'm not really sure.  @planetarium/libplanet Opinions?

Reply:
I think consistency is more important at this point. I'm sure most, if not all, of us are comfortable enough with using method syntax. Until we come up with some sort of guideline when to use query syntax and when to use method syntax, I think it is better to postpone mixing different types of syntaxes. 😶

Reply:
🆗